go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/cyphar/filepath-securejoin v0.2.3
 	github.com/docker/distribution v2.8.2+incompatible
 	github.com/docker/go-units v0.5.0
-	github.com/emicklei/go-restful/v3 v3.9.0
+	github.com/emicklei/go-restful/v3 v3.10.2
```

Comment:
I'm pretty sure this bump breaks things, see https://github.com/kubernetes/kubernetes/pull/115067

Reply:
I mean the [go-contrib instrumentation depends](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/v1.16.0/instrumentation/github.com/emicklei/go-restful/otelrestful/go.mod#L8) on that. ^^

I hope I can have a deeper look on Friday. At least their changelogs do not look suspicious.
https://github.com/emicklei/go-restful/blob/v3/CHANGES.md

Reply:
Whats the best way to debug this locally?

I did run the test locally and noticed that `TestCalculateLinuxResources` in `pkg/kubelet/kuberuntime` failed.
```diff
--- Expected
+++ Actual
@@ -9,3 +9,5 @@
  HugepageLimits: ([]*v1.HugepageLimit) <nil>,
- Unified: (map[string]string) <nil>,
+ Unified: (map[string]string) (len=1) {
+  (string) (len=16) "memory.oom.group": (string) (len=1) "1"
+ },
MemorySwapLimitInBytes: (int64) 0,
```

But in that case I assume its a local issue with my system settings :facepalm: 

https://github.com/kubernetes/kubernetes/blob/79c02ceb73f4d64f52a9d4c46785e4c7497493d9/pkg/kubelet/kuberuntime/kuberuntime_container_linux.go#L218-L226